### ISSUE TITLE
make test fails on linux x64

### ISSUE THREAD

**Issue by mikhailt** [VowpalWabbit/vowpal_wabbit#237]
> make test
> vw running test-suite...
> (cd test && ./RunTests -d -fe -E 0.001 ../vowpalwabbit/vw ../vowpalwabbit/vw)
> testing linux Testing vw: ../vowpalwabbit/vw
> Testing lda: ../vowpalwabbit/vw
> RunTests: '-D' to see any diff output
> RunTests: '-o' to force overwrite references
> RunTests: test 1: stderr OK
> RunTests: test 2: stderr OK
> RunTests: test 2: predict OK
> RunTests: test 3: stderr OK
> RunTests: test 4: stdout OK
> RunTests: test 4: stderr OK
> RunTests: test 5: minor (<0.001) precision differences ignored
> RunTests: test 5: stderr OK
> RunTests: test 6: stderr OK
> RunTests: test 6: predict OK
> RunTests: test 7: stderr OK
> RunTests: test 8: stderr OK
> RunTests: test 8: minor (<0.001) precision differences ignored
> RunTests: test 8: predict OK
> RunTests: test 9: stderr OK
> RunTests: test 9: predict OK
> RunTests: test 10: stderr OK
> RunTests: test 10: predict OK
> RunTests: test 11: stderr OK
> RunTests: test 12: stderr OK
> RunTests: test 13: stderr OK
> RunTests: test 14: stdout OK
> RunTests: test 14: stderr OK
> RunTests: test 15: stdout OK
> RunTests: test 15: stderr OK
> RunTests: test 16: stdout OK
> RunTests: test 16: stderr OK
> RunTests: test 17: minor (<0.001) precision differences ignored
> RunTests: test 17: stderr OK
> RunTests: test 18: stderr OK
> RunTests: test 19: stderr OK
> RunTests: test 20: stderr OK
> RunTests: test 20: predict OK
> RunTests: test 21: minor (<0.001) precision differences ignored
> RunTests: test 21: stderr OK
> RunTests: test 22: stdout OK
> RunTests: test 22: stderr OK
> RunTests: test 23: stdout OK
> RunTests: test 23: stderr OK
> --- diff -N --minimal --suppress-common-lines --ignore-all-space --strip-trailing-cr --side-by-side -W 160 train-sets/ref/active-simulation.t24.stderr stderr.tmp
> 0.923209   0.617612           83       613.4   1.0000   0.0494       46       | 0.923206   0.617608           83       613.4   1.0000   0.0494       46
> 0.843993   0.528494          169       767.4  unknown  -0.0771       87       | 0.843990   0.528514          169       767.4  unknown  -0.0771       87
> 0.772938   0.491751          261       961.3   1.0000   0.1576       13       | 0.772931   0.491736          261       961.3   1.0000   0.1576       13
> 0.688758   0.352542          430      1201.9   1.0000  -0.0663       18       | 0.688754   0.352555          430      1202.0   1.0000  -0.0663       18
> 0.614280   0.322631          569      1508.9   1.0000   0.3900       29       | 0.614280   0.322626          569      1508.9   1.0000   0.3900       29
> 0.540212   0.262029          796      1910.6  -1.0000  -0.5583       46       | 0.540212   0.262017          796      1910.6  -1.0000  -0.5583       46
> 0.469826   0.188841         1190      2389.2  unknown   0.3206       44       | 0.469827   0.188854         1190      2389.3  unknown   0.3206       44
> 0.430273   0.272207         1491      2987.1  -1.0000   0.1254       70       | 0.430278   0.272222         1491      2987.1  -1.0000   0.1254       70
> 0.376527   0.161817         1901      3734.8  unknown  -0.4625       48       | 0.376532   0.161816         1901      3734.8  unknown  -0.4625       48
> 0.370952   0.348673         2127      4669.3  unknown  -0.4482      179       | 0.370957   0.348679         2127      4669.4  unknown  -0.4482      179
> 0.359658   0.314489         2813      5836.9  unknown  -0.2033       35       | 0.359667   0.314513         2813      5836.9  unknown  -0.2033       35
> 0.328361   0.203584         3462      7300.9  -1.0000  -1.0000      151       | 0.328371   0.203595         3462      7300.8  -1.0000  -1.0000      151
> 0.296159   0.170245         4340      9168.1  -1.0000  -0.5148       70       | 0.296166   0.170237         4340      9168.0  -1.0000  -0.5148       70
> 0.279052   0.210640         5887     11460.5  unknown   0.5904       40       | 0.277338   0.202040         5933     11460.4  unknown   0.5531       81
> 0.255498   0.163368         6996     14390.5   1.0000   0.5009       49       | 0.257099   0.176171         7195     14326.4  unknown   0.2642      138
> 0.240619   0.181113         9174     17988.9  unknown  -0.7411       57       | 0.240224   0.172730         9786     17908.3  unknown  -0.3206       47
> weighted example sum = 19090.5                            | weighted example sum = 18393.3
> weighted label sum = -1355                            | weighted label sum = -1447.61
> average loss = 0.240509                               | average loss = 0.235951
> best constant = -0.136917                             | best constant = -0.157285
> total queries = 889                               | total queries = 884
> RunTests: test 24: FAILED: ref(train-sets/ref/active-simulation.t24.stderr) != stderr(stderr.tmp)
> make: **\* [test] Error 1
> 
> uname -a
> Linux xxxxx 3.2.0-58-generic #88-Ubuntu SMP Tue Dec 3 17:37:58 UTC 2013 x86_64 x86_64 x86_64 GNU/Linux
